src/crimson/osd/backfill_state.cc

```diff
@@ -278,6 +278,19 @@ BackfillState::Enqueuing::update_on_peers(const hobject_t& check)
   return result;
 }
 
+bool BackfillState::Enqueuing::Enqueuing::all_emptied(
+  const BackfillInterval& primary_bi,
```

Comment:
nit, s/primary_bi/primary_backfill_info/ or s/primary_bi/local_backfill_info/

Reply:
Switched to `local_backfill_info`. Thanks!